src/views/Logbook/LogbookDetail.tsx

```diff
@@ -1,19 +1,166 @@
+import classNames from "classnames"
+// import { useLocalization } from "gatsby-theme-i18n"
 import React, { useContext, useEffect } from "react"
 
+import env from "@/.env.json"
+import { Avatar, IconChevonLeft, Section } from "~/components"
 import { Container, LogbookContext, LogbookEditor, SEO } from "~/components"
 import Footer from "~/components/Layout/Footer"
 import Header from "~/components/Layout/Header"
+// import { Lang } from "~/enums"
+import { useResponsive } from "~/hooks"
+import { analytics, toEtherscanUrl } from "~/utils"
 
-// import * as styles from "./LogbookList.module.css"
+import * as styles from "./LogbookDetail.module.css"
 
 type PageProps = {
   id: string
   originalPath: string
 }
 
+type TokenIdProps = {
+  tokenId: string
+}
+
+type Log = {
+  sender: string 
+  message: string
+  createdAt: Date
+}
+
+type LogListProps = {
+  logs: Log[]
+}
+
+const LogListItem: React.FC<Log> = ({sender, message, createdAt}) => {
+  // const { locale } = useLocalization()
+  return (
+    <>
+    <li>
+      <article>
+        <div>
+          {message} 
+          i want an album of songs that each sounds like a radio-friendly hit, warm and crispy, but with math-rock-like polyrhythmic drum patterns instead of a regular beat. like a funhouse version of popular music that is accessible in every way except drums. like a crumbling teeth dream
+        </div>
+        <div className={styles.flexColumn}>
+          <div className={styles.columnItem}><a href={toEtherscanUrl(sender).url}>来自{sender.slice(0, 11)}...的宇宙訊號</a></div>
+          <div className={styles.columnItem}>{createdAt.toISOString().slice(0, 10)}</div>
+        </div>
+      </article>
+    </li>
+    <li>
```

Comment:
The duplicated entries and text here are only for the purpose to verify page layout. We will remove the them later.